Adds `teams cache remove` command. Closes #3205 (#3224)
pnp/cli-microsoft365: docs/docs/cmd/teams/cache/cache-remove.md

# teams cache remove

Removes the Microsoft Teams client cache

## Usage

```sh
m365 teams cache remove [options]
```

## Options

`--confirm`
: Don't prompt for confirmation

--8<-- "docs/cmd/_global.md"

## Remarks

> ### Clear the Microsoft Teams cache  

Reviewer: As it's the only heading in this section, let's remove it

This command will execute the following steps.

> - Stop the Microsoft Teams client.

Reviewer: For readability, let's merge it into one line:

```suggestion
- Stop the Microsoft Teams client. This will kill all the running `Teams.exe` tasks.
```

This will kill all the running `Teams.exe` tasks.

> - Clear the Microsoft Teams cached files.

Reviewer: For readability, let's move it into one line. Also, let's not mark OSes as code:

```suggestion
- Clear the Microsoft Teams cached files. For Windows it will delete all files and folders in the `%appdata%\Microsoft\Teams directory`. For macOS it will delete all files and folders in the `~/Library/Application Support/Microsoft/Teams` directory.
```

For `Windows` it will delete all files and folders in the %appdata%\Microsoft\Teams directory.
For `macOS` it will delete all files and folders in the  ~/Library/Application Support/Microsoft/Teams directory.

> !!! note

Reviewer: Let's make it stand out more

```suggestion
!!! important
```

    - You won't lose any user data by clearing the cache.
    - Restarting Teams after you clear the cache might take longer than usual because the Teams cache files have to be rebuilt.

> If the command is executed within the CLI Docker container, you will get the `Because you're running CLI for Microsoft 365 in a Docker container, we can't clear the cache on your host. Instead run this command on your host using 'npx ...'` error.

Reviewer: Let's simplify it by changing to active voice:

```suggestion
If you run the command in the CLI Docker container, you will get the following error message:

> Because you're running CLI for Microsoft 365 in a Docker container, we can't clear the cache on your host. Instead run this command on your host using 'npx ...'.
```

> If the command isn't executed from a Windows or MacOS system, you will get the `'abc' platform is unsupported for this command` error.

Reviewer: Let's simplify it by changing to active voice:

```suggestion
The command works only on Windows and macOS. If you run it on a different operating system, you will get the `'abc' platform is unsupported for this command` error.
```

## Examples

Removes the Microsoft Teams client cache

```sh
m365 teams cache remove
```

## More information

> - Guidance from the Microsoft Docs article: [Clear the Teams client cache](https://docs.microsoft.com/en-us/microsoftteams/troubleshoot/teams-administration/clear-teams-cache)

Reviewer: Let's simplify it to:

```suggestion
- [Clear the Teams client cache guidance](https://docs.microsoft.com/microsoftteams/troubleshoot/teams-administration/clear-teams-cache)
```
